### PR TITLE
webpack: now with babel

### PR DESCRIPTION
**Pull request by Pomax**
Makes it so that we don't have to worry about ES6 syntax anymore.

### CODE REVIEW

File: lib/teach-api.js [hunk 1]
```diff
@@ -122,7 +122,15 @@ _.extend(TeachAPI.prototype, {
...
   request: function(method, path) {
     if (emulateLogin) {
-      return { accept: function() { return { end: function(fn) { fn(false, { body: ''}); }}}};
+      return {
+        accept: function() {
+          return {
+            end: function(fn) {
+              fn(false, { body: ''});
+            }
+          };
```
Comment (Pomax) [inline lib/teach-api.js]: flagged by jscs/jshint

File: lib/resetreload.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ module.exports = {
   },
 
   reset: function() {
-    _reset = false
+    _reset = false;
```
Comment (Pomax) [inline lib/resetreload.js]: flagged by jscs/jshint

**Comment by Pomax** [comment]
@gvn thanks to the magic of "are you kidding me?" this branch works fine now - the timeout waiting for the index to rebuild apparently needs to be longer than 10 seconds on travis.

**Comment by gvn** [comment]
I'm seeing a console error on http://localhost:8008/en-US/

screen shot 2016-08-15 at 2 01 06 pm

**Comment by Pomax** [comment]
hm, and that's not there without these changes? (I didn't touch anything in the page-generate code paths)

**Comment by Pomax** [comment]
found it - for some reason page-generate was missing `var` for two variable assignments.

**Comment by gvn** [comment]
Yay! LGTM based on smoke testing & automated tests passing.

**Pull request merged**